adversarial_apps/src/components/searchbar.tsx

```diff
@@ -25,11 +26,19 @@ export default function SearchBar({ placeholder }: {placeholder: string}) {
     }, [searchTerm]);
 
     async function fetchResults(query: string) {
+        const data = { action: "obtain_cik_number", search_term: query };
+        const response = await fetch('/api/call-python-api', {
+          method: 'Post',
+          headers: {
+            'Content-Type': 'application/json'
+          },
+          body: JSON.stringify(data)
+        });
+        const result = await response.json();
+        console.log("Result is for:", result);
+        const companyNames = result.companies.map((company: { "Company Name": string }) => company["Company Name"]);
+        setResults(companyNames)
       // Mock results for demonstration; replace with actual fetch
```

Comment:
Remove // mock comments for demonstration line since no longer referencing code

Reply:
Fixed